CONTRIBUTING.md

```diff
@@ -0,0 +1,105 @@
+# Contributing
+
```

Comment:
We might want to also add a short tutorial on how to add a new adapter like we have in other libraries.

Reply:
I don't see it in other libraries. Can you point me to one?